.github/workflows/build.yml

```diff
@@ -101,6 +101,122 @@ jobs:
+          git init --quiet
+          git remote add origin https://github.com/${{ github.repository }}
+          git fetch --depth 1 origin ${{ github.sha }}
+          git checkout FETCH_HEAD
+      
+      - name: Setup environments
+        run: |
+          Set-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -Force
+          echo "DEPOT_TOOLS_WIN_TOOLCHAIN=0" >> $Env:GITHUB_ENV
+          echo "GYP_MSVS_OVERRIDE_PATH=C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise" >> $Env:GITHUB_ENV
+          echo "WINDOWSSDKDIR=C:\Program Files (x86)\Windows Kits\10" >> $Env:GITHUB_ENV
+
+      - name: Install depot_tools
+        run: |
+          Invoke-WebRequest -Uri https://storage.googleapis.com/chrome-infra/depot_tools.zip -OutFile depot_tools.zip
+          7z x -y -o"C:\workspace\depot_tools" .\depot_tools.zip
+          echo "C:\workspace\depot_tools" >> $Env:GITHUB_PATH
+
+      - name: GClient Sync
+        working-directory: C:\workspace\engine
+        run: |
+          gclient config --name=src\flutter --unmanaged https://github.com/flutter-tizen/engine
+          $env:PYTHONPATH="C:\workspace\depot_tools"
+          python3 src\flutter\ci\tizen\gclient-shallow-sync.py src\flutter\DEPS
+          gclient sync -v --no-history --shallow
+
+      - name: Build
+        working-directory: C:\workspace\engine\src
+        run: |
+          python3 .\flutter\tools\gn --target-os=linux --linux-cpu=${{ matrix.arch }} --no-goma --runtime-mode=${{ matrix.mode }}
+          ninja -C .\out\linux_${{ matrix.mode }}_${{ matrix.arch }} gen_snapshot
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: tizen-${{ matrix.arch }}-${{ matrix.mode }}_windows-x64
+          path: |
+            C:\workspace\engine\src\out\linux_${{ matrix.mode }}_${{ matrix.arch }}\gen_snapshot.exe
+
+  macos-build:
+    runs-on: macos-11
+
+    strategy:
+      matrix:
+        arch: [arm, arm64]
+        mode: [release, profile]
+
+    env:
+      OUTPUT_NAME: linux_${{ matrix.mode }}_${{ matrix.arch }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: src/flutter
+
+      - uses: actions/cache@v2
+        with:
+          path: src/out/${{ env.OUTPUT_NAME }}
+          key: out-macos-build-${{ env.OUTPUT_NAME }}-${{ github.sha }}
+          restore-keys: |
+            out-macos-build-${{ env.OUTPUT_NAME }}-
+
+      - name: Install depot_tools
+        run: |
+          git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+          echo "$PWD/depot_tools" >> $GITHUB_PATH
+      
+      - name: Install required packages
+        run: |
+          brew install md5sha1sum
+
+      - name: GClient Sync
+        run: |
+          gclient config --name=src/flutter --unmanaged https://github.com/flutter-tizen/engine
+          gclient setdep --var=download_android_deps=False --deps-file=src/flutter/DEPS
```

Comment:
Is this line only required for macos build?

Reply:
In Windows, the default value of `download_android_deps` is False.